### PR TITLE
testing: add files for linting, applied fixes for clang-tidy

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Use the Google style in this project.
```
Comment (gguuss) [inline .clang-format]: I'm unsure of whether it's necessary but consider adding Copyright to these configurations if it doesn't break anything

Comment (tmatsuo) [reply .clang-format]: Done

**Comment by tmatsuo** [comment]
@gguuss Thanks, PTAL

**Comment by tmatsuo** [comment]
@gguuss Ah, you already approved. I addressed your review comments. Submitting

**Pull request merged**